apps/aetx/src/aetx.erl

```diff
@@ -219,18 +221,25 @@ check(AeTx = #aetx{ cb = CB, tx = Tx }, Trees, Height, ConsensusVersion) ->
 check_from_contract(#aetx{ cb = CB, tx = Tx }, Trees, Height, ConsensusVersion) ->
     CB:check(Tx, aetx_contract, Trees, Height, ConsensusVersion).
 
--spec process(Tx :: tx(), Trees :: aec_trees:trees(), Height :: non_neg_integer(),
+
+-spec process_no_tx_hash(Tx :: tx(), Trees :: aec_trees:trees(), Height :: non_neg_integer(),
               ConsensusVersion :: non_neg_integer()) ->
     {ok, NewTrees :: aec_trees:trees()}.
-process(#aetx{ cb = CB, tx = Tx }, Trees, Height, ConsensusVersion) ->
-    CB:process(Tx, aetx_transaction, Trees, Height, ConsensusVersion).
+process_no_tx_hash(Tx, Trees, Height, ConsensusVersion) ->
+    process(Tx, Trees, Height, ConsensusVersion, <<>>).
```

Comment:
I am concerned about passing as no-value a value that can be misused easily as valid value. May I suggest a special value e.g. atom `no_tx_hash`?